automation/docker/Dockerfile

```diff
@@ -49,11 +54,32 @@ RUN sdkmanager --verbose "platform-tools" \
     "extras;google;m2repository"
 
 # Checkout source code
-RUN git clone https://github.com/mozilla-mobile/fenix.git
+#RUN git clone https://github.com/mozilla-mobile/fenix.git
+RUN git clone https://github.com/rpappalax/fenix.git -b dockerfile-add-test-deps --single-branch
 
 # Build project and run gradle tasks once to pull all dependencies
 WORKDIR /opt/fenix
-RUN ./gradlew --no-daemon assemble test lint
+RUN ./gradlew -PversionName=0.0 --no-daemon assemble test lint
```

Comment:
If you specifically `assembleDebug`, then you'll do a "machine local" build and won't need to provide the `-PversionName`.

Right now it fails the build because the build command used also builds nightlies and betas, and you require a version code to build nightlies or betas.

Hope that helps :)
(@pocmo so you're aware of the reasoning behind the `-PversionName` error)

Reply:
@mitchhentges yes, that works great.  I am, however, seeing a new error on Task :app:processArmDebugResources.  Any thoughts on this?  I've tried a number of things so far but no success
```
> Task :app:compileArmDebugKotlin
Unable to clear jar cache after compilation, maybe daemon is already down: java.rmi.ConnectException: Connection refused to host: 127.0.0.1; nested exception is:
	java.net.ConnectException: Connection refused (Connection refused)
Could not connect to kotlin daemon. Using fallback strategy.
Compilation with Kotlin compile daemon was not successful
java.rmi.UnmarshalException: Error unmarshaling return header; nested exception is:
	java.io.EOFException
	at sun.rmi.transport.StreamRemoteCall.executeCall(StreamRemoteCall.java:236)
	at sun.rmi.server.UnicastRef.invoke(UnicastRef.java:161)
	at java.rmi.server.RemoteObjectInvocationHandler.invokeRemoteMethod(RemoteObjectInvocationHandler.java:227)
	at java.rmi.server.RemoteObjectInvocationHandler.invoke(RemoteOb
```

Reply:
That's a great question, I'm not sure what's going on there.
You could try the mantra of getting a completely clean gradle build to see if that addresses it:
```
./gradlew clean
./gradlew --stop
./gradlew --no-daemon assembleDebug test lint
```

That's my only guess - other than that, [stackoverflow recommends something about temporary files](https://stackoverflow.com/a/54084016), but I'm out of guesses atm. This could be worth bringing up in #fenix in slack

Reply:
@mitchhentges I tried clean yesterday, but hadn't tried stop yet (though I saw someone else suggested that on SO).  thanks for the suggestions!